dockerfiles/ocaml-5.3.Dockerfile

```
@@ -0,0 +1,29 @@
+USER root
+
+# Install dune
+RUN opam install dune.3.19.0 --yes
+
+# Dune path is /home/opam/.opam/5.3/bin/dune
+ENV PATH="${OPAMROOT}/5.3/bin:${PATH}"
+
+WORKDIR /app
+
+# .git & README.md are unique per-repository. We ignore them on first copy to prevent cache misses
+COPY --exclude=.git --exclude=README.md . /app
```

Hadolint (2.12.0)

[error] invalid flag: --exclude

(DL1000)